pages/about/groups/task-forces/wcag2ict/work-statement.md

```diff
@@ -71,45 +76,47 @@ The following sections contain rough timelines for the WCAG2ICT work. Actual tim
+Phase 1 - Published update to [Guidance on Applying WCAG 2 to Non-web Information and Communications Technologies (WCAG2ICT) Group Note](https://www.w3.org/TR/wcag2ict-22/) (8 October 2024).
 
-Note: To include WCAG 2.2, the Recommendation version standard must be published with sufficient time to update the draft WCAG2ICT Note with finalized requirements verbiage.
+Goals were to incorporate new WCAG 2.1 and 2.2 Level A and AA success criteria and making other necessary updates to the other content in the WCAG2ICT document to align with WCAG 2.2 have been completed. This phase also addressed known documented issues.
 
-*   April - August 2022:
+*   Plan: April - August 2022, Actual: April - August 2022
     *   obtain approval from AG WG and launch the WCAG2ICT Task Force;
     *   recruit TF participants with necessary expertise.
-*   September 2022:
-    *   develop a brief requirements document and obtain agreement from AG WG;
-*   October 2022 - April 2023:
+*   Plan: September 2022, Actual: October 2022 
+    *   develop a brief requirements document (as inclurporated into the work statement) and obtain agreement from AG WG ;
+*   Plan: October 2022 - April 2023, Actual: October 2022 - August 2023 
     *   develop an initial draft with updates to the Working Group Note;
     *   develop initial discussion drafts of remaining sections of the Note;
     *   develop initial discussion drafts of Note sections;
-    *   start reporting to the AG WG (schedule and frequency TBD)
-*   July - June 2023:
+    *   start reporting to the AG WG (schedule and frequency aligned to when groups of SCs are added)
+*   Plan: July - June 2023, Actual: August 2023 - July 2024:
     *   incorporate first feedback from TF and AG WG into a (publicly visible) Editors’ Draft;
     *   publish Draft of the updated Working Group Note through the AG WG;
     *   start processing feedback on draft.
-*   July - October 2023:
+*   Plan: July - October 2023, Actual: July - October 2024
     *   process feedback on document, further develop and edit WG Note;
     *   publish updated formal draft, process comments.
-*   November 2023:
+*   Plan: November 2023, Actual: November 2024:
     *   Publish updated WCAG2ICT Note.
 
 ### Phase 2
 
-The next phase of updates would include any other in-scope work, including Level AAA, that was not addressed in Phase 1. Details of the Phase 2 timeline will be developed outside of this document when more is known about the scope of work left to do.
+The next phase of updates will include any other in-scope work, including:
+* Addition of guidance for Level AAA success criteria. Details of the Phase 2 timeline are found outside of this document. See the [schedule and milestones](https://github.com/w3c/wcag2ict/wiki/Schedule-and-milestones) page in the GitHub WCAG2ICT Wiki.
```

Comment:
@daniel-montalvo per last week's meeting notes, should this be changed to be further down the list of items in the list?

Reply:
Yes, I think so, as per https://www.w3.org/2024/11/21-wcag2ict-minutes#f10a